script/LaunchProjectFor.s.sol

```diff
@@ -1,43 +1,42 @@
 /**
  * DEPRECATED - Use jbm for now
```

Comment:
Not sure what we should do with this (this is a deploy script, not a test)
@mejango did u use this recently? Maybe we'll need it for some test deployment on Sepolia (if any?)

Reply:
delete